mobify/progressive-web-scaffold
# Add buddybuild prebuild step

(brief description / PR title)

 **JIRA**: https://mobify.atlassian.net/browse/HYB-980

## Changes
- Add `buddybuild_prebuild.sh`

## How to test-drive this PR
- Look at https://dashboard.buddybuild.com/apps/5877dde0dc8b3501004cb0a6 and look at the logs to make sure the prebuild step is being run
